pyop2/types/set.py

```diff
@@ -424,11 +418,13 @@ def _kernel_args_(self):
     def _argtypes_(self):
         return self._superset._argtypes_ + (ctypes.c_voidp, )
 
-    # Look up any unspecified attributes on the _set.
-    def __getattr__(self, name):
-        """Returns a :class:`Set` specific attribute."""
-        value = getattr(self._superset, name)
-        return value
+    @property
+    def _parent(self):
+        return self._superset
```

Comment:
Shouldn't this be `self._superset._parent`?

Reply:
No, because the superset could also be a Set and the class `Set` don't have the attribute _parent. But I looked into it and `_parent` is used to get the attribute `name` in a specific call `self._parent.name`maybe inherited from the `ExtrudedSet` but not sure. I can change it to 
```
@property
 def name(self):
     return self._superset.name
```
I think this makes more sense. We then need to run the CI again to make sure there isn't another attribute called over this `_parent` attribute.

Reply:
Yeah that makes more sense. Go for it.

Reply:
You can now run the Firedrake CI once more.